lib/Controller/PageController.php

```diff
@@ -999,6 +1004,16 @@ public function showForm($render, $uid, $pageId) {
             $ft = $this->l->t('Book Your Appointment');
         }
 
+        $hasActiveValidSession = false;
+        $clientList = [];
+
+        if($this->userSession->getUser()){
+            $hasActiveValidSession = $this->userSession->getUser()->getUid() == $uid;
```

Comment:
I didn't quite understand. This part of the code checks if the current user has the same id of the owner of the appointments page, if so, it gets their clients, otherwise it won't fetch any clients.

Reply:
You're completely correct, that's my bad! Though one change, you should do a strict comparison, so using `===` instead of `==`. You generally never want to use `==`.